blazor/datagrid/in-line-editing.md

```diff
@@ -1269,3 +1269,24 @@ public class OrderDetails
 {% endtabs %}
 
 {% previewsample "https://blazorplayground.syncfusion.com/embed/LthINCtgBmWlAIwy?appbar=false&editor=false&result=true&errorlist=false&theme=bootstrap5" %}
+
+## Supported events for inline and dialog editing
+
+The Syncfusion Blazor DataGrid triggers the following events during `Inline` and `Dialog` editing:
```

Comment:
Inline and dialog editing in the Syncfusion Blazor DataGrid facilitate tailored ways to modify records, catering to different user needs. This section outlines the key events associated with these editing modes, providing essential information to effectively implement and customize the grid's functionality for enhanced user interaction. --- Ensure once in Continue AI by feeding the entire UG documentation and draft this content based on that